packages/protocol/contracts/libs/LibInvalidTxList.sol

```diff
@@ -47,7 +47,7 @@ library LibInvalidTxList {
         TX_GAS_LIMIT_TOO_SMALL
```

Comment:
So if we remove the assertion, is the `OK` enum option is also unnecessary?

Reply:
I think so

Reply:
yes, it can be removed.